doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -304,6 +304,10 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.inf.ntw.08` | Infrastructure | Network | The Architecture **must**  provide a default network which implements the Kubernetes network model. | |
 | `req.inf.ntw.09` | Infrastructure | Network | The networking solution **must not** interfere with or cause interference to any interface or network it does not own. | |
 | `req.inf.ntw.10` | Infrastructure | Network | The Architecture **must** support cluster wide coordination of IP address assignment. | |
+| `req.inf.ntw.11` | Infrastructure 	Network 	The Architecture must support multiple network interfaces to be configured for a pod. | |
+| `req.inf.ntw.12` | Infrastructure 	Network 	The networking solution to support req.inf.ntw.11 must conform to the Kubernetes Networking CRD de-facto standard (K8S Network Plumbing Working Group). | |
```

Comment:
Could you provide a link to the documentation of the standard please?

Reply:
@tomkivlin Added. Thanks